kalastatic.module

```diff
@@ -176,6 +176,9 @@ function kalastatic_get_drupal_asset($type) {
 function kalastatic_asset_deliver($assets) {
   drupal_add_http_header('Content-Type', 'application/json; charset=utf-8');
```

Comment:
@soniktrooth The other thing I noticed was that while we're outputting HTML here, its content type is set to JSON. Think we should remove this line?